spec/bcrypt/engine_spec.rb

```diff
@@ -12,8 +12,11 @@
 
 describe "The BCrypt engine" do
   specify "should calculate the optimal cost factor to fit in a specific time" do
-    first = BCrypt::Engine.calibrate(100)
-    second = BCrypt::Engine.calibrate(400)
+    start_time = Time.now
+    BCrypt::Password.create("testing testing", :cost => BCrypt::Engine::MIN_COST + 1)
+    min_time_ms = (Time.now - start_time) * 1000
+    first = BCrypt::Engine.calibrate(min_time_ms)
+    second = BCrypt::Engine.calibrate(min_time_ms * 2)
```

Comment:
Dropping this to 2x instead of the original 4x made the tests start failing on Windows now.  Moved back to 4x in https://github.com/codahale/bcrypt-ruby/commit/fc652e5248a4132af2c5f5c0b61eeceff02f4316